docs/source/model/signatures.rst

```diff
@@ -748,7 +748,51 @@ model signatures in log_model calls when signatures aren't specified.
     Secondly, and just as importantly, it **validates the model's requirements**. This input example is utilized to execute a prediction 
     using the model that is about to be logged, thereby enhancing the accuracy in identifying model requirement dependencies.
     It is **highly recommended** to always include an input example along with your models when you log them.
+    Since MLflow 2.16.0, if the provided input example fails on validation, a warning is issued with an instruction to validate the input example
+    prior to model deployment.
 
+Since MLflow 2.16.0, when logging a model with an input example, there are two files saved into the model's artifacts directory:
+
+- ``input_example.json``: The input example in JSON format.
+- ``serving_input_payload.json``: The input example converted to JSON format for querying a deployed model endpoint.
+
+The following example demonstrates the difference between the two files:
+
+.. code-block:: python
+
+    import mlflow
+
+
+    class MyModel(mlflow.pyfunc.PythonModel):
+        def predict(self, context, model_input, params=None):
+            return model_input
+
+
+    with mlflow.start_run():
+        model_info = mlflow.pyfunc.log_model(
+            python_model=MyModel(),
+            artifact_path="model",
+            input_example={"question": "What is MLflow?"},
+        )
+
+Example files:
+
+ +---------------------------------------+-----------------------------------------+
+ |  input_example.json                   |  serving_input_payload.json             |
+ +---------------------------------------+-----------------------------------------+
```

Comment:
Not a blocker, but I'd vertically stack the examples for editability

```
.. code-block::
   
     # input_example.json

.. code-block::
   
     # serving_input_payload.json
```

Reply:
I think we can use [list-table](https://github.com/mlflow/mlflow/blob/d828b72897415548601e9825c43485e13136c96f/docs/source/llms/transformers/task.rst?plain=1#L77), much easier to edit imo

Reply:
Updated 
<img width="1343" alt="image" src="https://github.com/user-attachments/assets/8b4f3358-1687-4276-ba5c-d7997f4d2f4d">